### PR TITLE
Gregg's NBA Challenge

### PR DESCRIPTION
**Pull request by NYMets89**

### CODE REVIEW

File: gregg/nba_challenge.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# TODO: As above, print out the paragraph with only 1 sentence per line, and all in upper case
+print(('The Lakers went all in this offseason and swung a deal for former Pelicans forward Anthony Davis. \nThey sent a package of Brandon Ingram, Lonzo Ball, Josh Hart, and 3 first-round picks to New Orleans to land Davis. \nThose three have made good developments with the Pelicans, especially Brandon Ingram. But, the deal is still a huge win for the Lakers as Lebron, Davis, and company have put together an incredible season. \nLos Angeles has ridden James and Davis, along with a supporting cast built around them, to the second-best record in the NBA. \nThe Lakers ended the season atop the Western Conference with a record of 49-14. \nThey were narrowly behind the Bucks for the best record in the league. \nDavis proved to the final piece necessary for the Lakers to rebound from missing the playoffís last year. \nLos Angeles was a dominant club on both sides of the ball and are in a position to have another successful year next season').upper())
+
+print('Challenge 3.3: Make a boolean variable indicating whether you think the Lakers are the best NBA team')
```
Comment (liubrandon) [inline gregg/nba_challenge.py:51]: Remember you can break up lines into multiple lines with \

File: gregg/nba_challenge.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+print('Challenge 3.1: Print out the paragraph but with only 1 sentence per line')
+# TODO: Print the giant chunk of text out using escape characters so each sentence comes out on a new line
+print('The Lakers went all in this offseason and swung a deal for former Pelicans forward Anthony Davis. \nThey sent a package of Brandon Ingram, Lonzo Ball, Josh Hart, and 3 first-round picks to New Orleans to land Davis. \nThose three have made good developments with the Pelicans, especially Brandon Ingram. But, the deal is still a huge win for the Lakers as Lebron, Davis, and company have put together an incredible season. \nLos Angeles has ridden James and Davis, along with a supporting cast built around them, to the second-best record in the NBA. \nThe Lakers ended the season atop the Western Conference with a record of 49-14. \nThey were narrowly behind the Bucks for the best record in the league. \nDavis proved to the final piece necessary for the Lakers to rebound from missing the playoffís last year. \nLos Angeles was a dominant club on both sides of the ball and are in a position to have another successful year next season.')
```
Comment (liubrandon) [inline gregg/nba_challenge.py:45]: For long lines of code, it's recommended to break them up into shorter ones using \

**Comment by NYMets89** [comment]
I shortened the code per your comments

**Review by liubrandon — APPROVED** [review]
looks good!

**Pull request merged**